### DISCUSSION TITLE
Hubs cloud is currently offline.

### DISCUSSION THREAD

**Discussion by nupur711** [mozilla/hubs#4900] [Hubs Cloud Questions]
After successful deployment of the custom client, the main page of hubs cloud shows following error
![ss5](https://user-images.githubusercontent.com/28728455/145202382-37ba976a-ca14-41a6-b3e0-d58713502f21.png)

**Comment by pattersonbl2** [comment]
It's very hard to tell what could be the issue based on the screenshot.  This could range from service on the server not working to a database issue.  I would need more logs information to start.

**Reply by nupur711** [reply]
![ss6](https://user-images.githubusercontent.com/28728455/145354249-15ab95df-273f-45fa-8dab-6d4a3499af03.png)
This is what I am getting after deployment of the custom client on AWS hubs cloud.

**Comment by esperecyan** [comment]
I am having the same problem in my environment.

1. `npm run deploy`
2. When I access the top page, it is now loading for a few seconds and then Offline is displayed.
3. `npm run undeploy`
4. Still Offline.
5. Switched the stack to Offline and then to Online. Restarted the EC2 instance.
6. Offline is now displayed as soon as I open the top page.

If I try `npm run undeploy` in this state, I will see the following:
> \> hubs@0.0.1 undeploy
> \> node -r @babel/register -r esm -r ./scripts/shim scripts/undeploy.js
> 
> Browserslist: caniuse-lite is outdated. Please run:
> npx browserslist@latest --update-db
> √ hubs.example now using Hubs master client.

If I try `npm run deploy` in this state, I will get the following error:
> \> hubs@0.0.1 deploy
> \> node -r @babel/register -r esm -r ./scripts/shim scripts/deploy.js
> 
> Browserslist: caniuse-lite is outdated. Please run:
> npx browserslist@latest --update-db
> Deploying to hubs.example.
> &nbsp;&nbsp;| 
> ○○○○○○○○○\hubs\node_modules\node-fetch\lib\index.js:272
> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;return Body.Promise.reject(new FetchError(\`invalid json response body at ${_this2.url} reason: ${err.message}`, 'invalid-json'));
> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;^
> FetchError: invalid json response body at https://hubs.example/api/ita/configs/hubs reason: Unexpected token < in JSON at position 0
> &nbsp;&nbsp;&nbsp;&nbsp;at ○○○○○○○○○\hubs\node_modules\node-fetch\lib\index.js:272:32
> &nbsp;&nbsp;&nbsp;&nbsp;at processTicksAndRejections (node:internal/process/task_queues:96:5) {
> &nbsp;&nbsp;type: 'invalid-json'
> }
> 
> Node.js v17.1.0
